src/Features/CSharp/Portable/Completion/CompletionProviders/SnippetCompletionProvider.cs

```diff
@@ -41,6 +41,8 @@ internal override bool IsInsertionTrigger(SourceText text, int characterPosition
             return CompletionUtilities.IsTriggerCharacter(text, characterPosition, options);
         }
 
+        internal override ImmutableHashSet<char> PossibleTriggerCharacters => CompletionUtilities.PossibleTriggerCharacters;
```

Comment:
i wouldn't call the member on the completoin provider PossibleTriggerCharacters.  They *are* the trigger characters, right?